app/assets/stylesheets/print.scss

```diff
@@ -32,4 +32,10 @@
     display: none !important; // In no circumstances do we want buttons to appear when the page is printed,
                               // regardless of class-based button styles
   }
-}
\ No newline at end of file
+}
+// Grover/pupetteer has a bug https://github.com/puppeteer/puppeteer/issues/4125
+// this ensures that the bug does not cause the titles in H1 and H2 elements to
+// split on the characters 'fi'
+* {
```

Comment:
We might want to make it so that this file is only included in the print layout, because at the moment this will affect every element on every screen